From 80e43f6b8c8316075d8f9f6553ad271e168060b1 Mon Sep 17 00:00:00 2001
From: Chris Lumens <clumens@redhat.com>
Date: Wed, 22 Jul 2015 09:13:11 -0400
Subject: [PATCH 09/55] Add a new makefile target that does everything needed
 for jenkins.

We should have as much of the logic of how the CI tests are run in source
control as possible, so that's what this target is for.  Besides this, jenkins
just runs a "git clean" first.
---
 Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Makefile b/Makefile
index ac144046133..ade4421aa25 100644
--- a/Makefile
+++ b/Makefile
@@ -84,3 +84,4 @@ archive:
 upload: archive
 	@scp grubby-$(VERSION).tar.bz2 fedorahosted.org:grubby
 
+ci: test
-- 
2.17.1

